s3_meta_sync.gemspec

```diff
@@ -8,6 +8,6 @@ Gem::Specification.new name, S3MetaSync::VERSION do |s|
   s.homepage = "https://github.com/grosser/#{name}"
   s.files = `git ls-files lib/ bin/`.split("\n")
   s.license = "MIT"
-  s.add_runtime_dependency "aws-sdk-core", '~> 2.0'
+  s.add_runtime_dependency "aws-sdk", '~> 3.0'
```

Comment:
should be `"aws-sdk-s3", "~> 1.0"` as per https://rubygems.org/gems/aws-sdk-s3

Reply:
I tried this and it didn't work with Classic since it relies on Classic core... any suggestions?